Tasks/PackerBuildV0/package.json

```diff
@@ -5,9 +5,9 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^16.11.39",
     "@types/q": "^1.0.7",
-    "azure-pipelines-task-lib": "^4.3.1",
-    "azure-pipelines-tasks-azure-arm-rest": "3.217.1",
     "agent-base": "^6.0.2",
+    "azure-pipelines-task-lib": "^4.13.0",
+    "azure-pipelines-tasks-azure-arm-rest": "3.217.1",
```

Comment:
Why not use the latest version? it's 3.241.2

Reply:
updated to latest version 3.241.2